### PR TITLE
PDI-15038 Support MongoDB AuthenticationDatabase Parameter in MongoDB…

### PR DESCRIPTION
**Pull request by flatrixnet**
… Steps and Support the SCRAM-SHA-1 security

### CODE REVIEW

**Comment by pamval** [comment]
Needs to be rebased

**Comment by flatrixnet** [comment]
Rebased
Can you please confirm that is everything ok?

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   </parent>
   <groupId>pentaho</groupId>
   <artifactId>pentaho-mongodb-plugin</artifactId>
-  <version>7.0-SNAPSHOT</version>
+  <version>6.1-SNAPSHOT</version>
```
Comment (pamval) [inline pom.xml]: This is not correct. Version should be 7.0-SNAPSHOT

Comment (flatrixnet) [inline pom.xml]: Fixed

**Comment by pamval** [comment]
Done in #124

**Pull request closed without merging**